tests/e2e/sendEmail.test.ts

```diff
@@ -64,7 +69,7 @@ describe('web3mail.sendEmail()', () => {
       const sendEmailResponse = await web3mail.sendEmail(params);
       expect(sendEmailResponse.taskId).toBeDefined();
     },
-    3 * MAX_EXPECTED_BLOCKTIME
+    MAX_EXPECTED_WEB2_SERVICES_TIME
```

Comment:
`sendEmail` actually sends a `matchOrders` tx under the hood we must take this time into account.
```suggestion
    2 * MAX_EXPECTED_BLOCKTIME + MAX_EXPECTED_WEB2_SERVICES_TIME
```